migrations/versions/307_delete_dm_datetime.py

```diff
@@ -0,0 +1,43 @@
+"""
+
+Revision ID: 307
```

Comment:
filename should be `0307` so that it's alphabetical in the file directory.


Also, the revision in code should be the same as the filename - we compare the alembic version in the database (defined by `revision`) to the files present on disk when deciding whether to apply database migrations during deploys. see https://github.com/alphagov/notifications-api/blob/master/scripts/check_if_new_migration.py. we could probably avoid this quirk by calling `flask db heads`, but still, it's good to have a descriptive name thats easy to search for on disk.

Reply:
Thanks, believe this is addressed now